core/src/HIP/Kokkos_HIP_Abort.hpp

```diff
@@ -50,18 +50,6 @@
 
 #include <hip/hip_runtime.h>
 
-// FIXME_HIP ROCm 4.5 version header include would be <rocm/rocm_version.h>
-#if __has_include(<rocm_version.h>)
-#include <rocm_version.h>
-#define KOKKOS_IMPL_ROCM_VERSION \
-  ROCM_VERSION_MAJOR * 10000 + ROCM_VERSION_MINOR * 100 + ROCM_VERSION_PATCH
```

Comment:
@Rombur you might want to include this in `HIP::print_configuration()` (see #5419 )

Reply:
If so, it would probably more sense to have it somewhere else, right?

Reply:
Yes